### PR TITLE
[wip] chore: update launchpad for dibbler

### PR DESCRIPTION
**Pull request by philipr-za**
Description
---
Some fixes to the launchpad docker files to get them working under dibbler

### CODE REVIEW

File: applications/launchpad/docker_rig/docker-compose.yml [hunk 1]
```diff
@@ -25,16 +25,16 @@ services:
       APP_EXEC: tari_base_node
       WAIT_FOR_TOR: ${WAIT_FOR_TOR:-0}
       TARI_NETWORK: ${TARI_NETWORK}
-      TARI_BASE_NODE__WEATHERWAX__TOR_CONTROL_AUTH: "password=${TOR_CONTROL_AUTH:-tari}"
-      TARI_BASE_NODE__WEATHERWAX__DATA_DIR: "/blockchain/weatherwax"
+      TARI_BASE_NODE__DIBBLER__TOR_CONTROL_AUTH: "password=${TOR_CONTROL_AUTH:-tari}"
+      TARI_BASE_NODE__DIBBLER__DATA_DIR: "/blockchain/weatherwax"
```
Comment (delta1) [inline applications/launchpad/docker_rig/docker-compose.yml:29]: In the new config file I think this is changed to 

```suggestion
      TARI_BASE_NODE__DIBBLER__DATA_DIR: "/dibbler"
```

File: applications/launchpad/docker_rig/docker-compose.yml [hunk 2]
```diff
@@ -64,12 +64,12 @@ services:
       TERM: "linux"
       TARI_WALLET_PASSWORD: ${TARI_WALLET_PASSWORD:-tari}
       TARI_NETWORK: ${TARI_NETWORK:-weatherwax}
```
Comment (delta1) [inline applications/launchpad/docker_rig/docker-compose.yml:66]:
```suggestion
      TARI_NETWORK: ${TARI_NETWORK:-dibbler}
```

File: applications/launchpad/docker_rig/base_node.Dockerfile [hunk 3]
```diff
@@ -47,10 +48,10 @@ RUN apt update && apt -y install \
 
 RUN groupadd -g 1000 tari && useradd -s /bin/bash -u 1000 -g 1000 tari
 
-RUN mkdir -p "/var/tari/base_node/weatherwax" &&
-  mkdir -p "/var/tari/base_node/igor" &&
-  mkdir -p "/var/tari/base_node/mainnet" &&
-  chown -R tari.tari "/var/tari/base_node"
+RUN mkdir -p "/var/tari/base_node/weatherwax" \
```
Comment (delta1) [inline applications/launchpad/docker_rig/base_node.Dockerfile:51]:
```suggestion
RUN mkdir -p "/var/tari/base_node/dibbler" \
```

**Review by delta1 — COMMENTED** [review]
Looking good, I know it's WIP just spotted some s/weatherwax/dibbler/

**Comment by CjS77** [comment]
Can you submit further updates into #3769

**Pull request closed without merging**